services/brig/src/Brig/API/User.hs

```diff
@@ -849,57 +936,51 @@ revokeIdentity key = do
 -- Change Account Status
 
 changeAccountStatus ::
-  forall m.
-  ( MonadClient m,
-    MonadLogger m,
-    MonadIndexIO m,
-    MonadReader Env m,
-    MonadMask m,
-    MonadHttp m,
-    HasRequestId m,
-    MonadUnliftIO m
+  forall r.
+  ( Member (Embed HttpClientIO) r,
+    Member NotificationSubsystem r,
+    Member Async r,
+    Member (Concurrency 'Unsafe) r,
+    Member TinyLog r
   ) =>
   List1 UserId ->
   AccountStatus ->
-  ExceptT AccountStatusError m ()
+  ExceptT AccountStatusError (AppT r) ()
 changeAccountStatus usrs status = do
   ev <- mkUserEvent usrs status
-  lift $ mapConcurrently_ (update ev) usrs
+  lift $ liftSem $ unsafePooledMapConcurrentlyN_ 16 (update ev) usrs
```

Comment:
`mapConcurrently_` (the non-pooled version) can be captured with the `Async` effect, it's something like `sequenceConcurrently . fmap` 
https://hackage.haskell.org/package/polysemy-1.9.1.3/docs/Polysemy-Async.html#v:sequenceConcurrently

Reply:
I find it a bit odd to do this without pooling because this can be called when all users of a team are to be suspended. So, we shouldn't be doing this anyway.